lib/scrabble/scrabble-class.rb

```diff
@@ -0,0 +1,61 @@
+module Scrabble
+  class Scrabble
+    SCORES = {
+      1 => ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+      2 => ["d", "g"],
+      3 => ["b", "c", "m", "p"],
+      4 => ["f", "h", "v", "w", "y"],
+      5 => ["k"],
+      8 =>["j", "x"],
+      10 => ["q", "z"]
+    }
+
+    def self.score(word)
+      # Edge case - no input, less than 8 letters
+      if word == "" || word == nil || word.length > 7
+        return "ERROR"
+      end
+
+      # Starts word_score at zero
+      word_score = 0
+      word.downcase!
+
+      # Count points letter by letter
+      word.each_char do |char|
+        # Edge case - no numbers/spaces/weird things in your inputed word
+        if (char >= "a") && (char <= "z")
```

Comment:
Strings implement Comparable, which has a method called `#between?`, so you could do this:
```
if char.between?('a', 'z')
```